overlays/openstack-lb-overlay.yaml

```diff
@@ -1,13 +1,25 @@
 applications:
-  kubeapi-load-balancer: null
+  kubeapi-load-balancer: null                            # excludes the kubeapi-load-balancer
+  kubernetes-control-plane:
+    options:
+      allow-privileged: "true"
   openstack-integrator:
     annotations:
-      gui-x: "600"
-      gui-y: "300"
     charm: openstack-integrator
     num_units: 1
     trust: true
+  openstack-cloud-controller:
+    charm: openstack-integrator
+    channel: edge
```

Comment:
Is it safe to point to the edge channel?

Reply:
uh... no let's fix that...